scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -83,6 +84,20 @@ public static BuildTargetResult PublishSdkInstallerFile(BuildTargetContext c)
         }
 
         [Target]
+        [BuildPlatforms(BuildPlatform.Windows, BuildPlatform.OSX)]//, BuildPlatform.Ubuntu)]
+        public static BuildTargetResult PublishSharedFxInstallerFile(BuildTargetContext c)
+        {
+            var installerFile = c.BuildContext.Get<string>("CombinedFrameworkHostInstallerFile");
```

Comment:
Should the name of the function match what is being published here?

`PublishCombinedFrameworkHostInstallerFile`